<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperationTab</class>
 <widget class="QWidget" name="OperationTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OperationTab</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="operateButton">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>go</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="SignalDisplayWidget" name="leftSignal" native="true"/>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="SignalDisplayWidget" name="rightSignal" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SignalDisplayWidget</class>
   <extends>QWidget</extends>
   <header>signal_display_widget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
